src/UI/Component/Panel/Factory.php

```diff
@@ -113,6 +116,11 @@ public function report($title,$sub_panels);
 	 *      Report Panels: >
 	 *        Report Panels contain sections as Sub Panels each displaying
 	 *        different aspects of one item.
+	 *      Presentation Table: >
+	 *        Use Presentation Tables to make data explorable that does not
+	 *        correspond to a single item or if the purpose of your listing is
+	 *        clearly explorational rather than representational.
```

Comment:
Please indicate how "explorational" is to be distinguished from "representational". We need a criterion to be able to decide what to collection to use for what case. 

Again please drop the word "table" if you talk about a listing. These terms will be presented to third parties and they should be able to understand what is what and why this is one family and not in another.

Reply:
I removed that "representational", clearly missworded.

I'm still convinced that is correct to talk about a table here.